android/sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1250,6 +1267,35 @@ public void uncaughtException(Thread t, Throwable e) {
     @SuppressLint({"NewApi", "MissingPermission"})
     @Override
     public int onStartCommand(Intent intent, int flags, int startId) {
+        if (intent != null && intent.hasExtra(TransportConstants.VEHICLE_INFO)) {
+            receivedVehicleType = new VehicleType(
+                    (HashMap<String, Object>) intent.getSerializableExtra(TransportConstants.VEHICLE_INFO)
+            );
+        }
+        if (firstStart) { // So we only run logic from this classes onCreate once
+            firstStart = false;
+            if (!initCheck()) { // Run checks on process and permissions
```

Comment:
This code is copied form `SdlRouterService.onCreate()`. You need to remove it from `onCreate()`

```
  if (!initCheck()) { // Run checks on process and permissions
                deployNextRouterService();
                closeSelf();
            }
            initPassed = true;


            synchronized (REGISTERED_APPS_LOCK) {
                registeredApps = new HashMap<String, RegisteredApp>();
            }
            closing = false;

            synchronized (SESSION_LOCK) {
                this.bluetoothSessionMap = new SparseArray<String>();
                this.sessionHashIdMap = new SparseIntArray();
                this.cleanedSessionMap = new SparseIntArray();
            }

            packetExecutor = Executors.newSingleThreadExecutor();

            startUpSequence();
```